README.md

```diff
@@ -17,7 +17,7 @@ This extension assumes you have the following binaries on your `PATH`:
    * `draft` (optional)
    * `minikube` (optional)
 
-For `kubectl`, `helm` and `draft` the binaries need not be on the system PATH, provided you tell the extension their locations using the appropriate `vs-kubernetes -> vs-kubernetes.${tool}-path` configuration setting.  See [Extension Settings](#extension-settings) below.
+For `kubectl`, `helm` and `draft` the binaries need not to be on the system PATH. You tell to the extension their locations using the appropriate `vs-kubernetes -> vs-kubernetes.${tool}-path` configuration setting.  See [Extension Settings](#extension-settings) below.
```

Comment:
In the first sentence, please remove the added 'to' - should read 'the binaries need not be...'.  Or (perhaps better) change it to 'the binaries do not need to be...'